js/src/components/picc-meter.js

```diff
@@ -0,0 +1,193 @@
+var CLASS_PREFIX = 'picc-meter-';
+
+window.PICCMeter = document.registerElement('picc-meter', {
```

Comment:
Or is it being global a requirement of it being a web component? I could also imagine index.js requiring all of the components and adding them to the global object there. Maybe I'll change my mind as a read through tthis more.